packages/addons/service/nut/package.mk

```diff
@@ -0,0 +1,57 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="nut"
+PKG_VERSION="2.8.0"
+PKG_REV="100"
+PKG_SHA256="c3e5a708da797b7c70b653d37b1206a000fcb503b85519fe4cdf6353f792bfe5"
+PKG_ARCH="any"
+PKG_LICENSE="GPLv2+"
+PKG_SITE="https://github.com/networkupstools/nut"
+PKG_URL="https://github.com/networkupstools/${PKG_NAME}/releases/download/v${PKG_VERSION}/nut-${PKG_VERSION}.tar.gz"
+PKG_MAINTAINER="Viktor Drobot <linux776@gmail.com> (dvdesolve)"
```

Comment:
I'd suggest dropping the email address unless you want extra spam and fanmail (but up to you)

Reply:
I think it will be ok because my mail can be found almost in each Arch package that I maintain